### ISSUE TITLE
Multiple colonies per player

### ISSUE THREAD

**Issue by Chinthor** [ldtteam/minecolonies-features#711]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my feature does not already exist.
- [x] I made sure that **this feature request is not a duplicate of any existing feature request.**

### Context 

- Minecraft Version: 1.18
- MineColonies Version:  minecolonies-1.18.2-1.0.919-ALPHA

## Is your feature request related to a problem?

Just that as a creative SSP, or SMP server owner, I'd like to get a colony up and running, self sustaining, and then move on and build a different one in a different style in a different biome. Preferably without killing all the first colonists I trained up.

## Feature description

Allow multiple colonies to be active per player. The system should assume that any colony command or action done by the colony owner is applied to the colony he is standing in at the moment. Any command executed outside all colony borders would need to specify a colony ID. This should be implemented at the multiplayer level. If the server can operate multiple colonies that are each overseen by a different player, it should have no issues doing the same management for the same number of colonies all owned by one player.

### Notes or related things

This could also facilitate quest/adventure/exploration/economy servers where the server owner manages all colonies, players cannot interfere with them, merely interact with them. One system that an admin could set up: Colonies will produce resources and periodically deliver batches of them to publicly available inventories. Pumping building materials into a colony will lead to building upgrades and higher quality/faster resource production. Certain buildings are planned out but not built. Build request triggered by delivering a quest item to the colony.

To be clear, not saying any of these quest bits NEED to be added by MineColonies. Just that they are a small sample of what could be done.
---
## Viewers

* Add a thumbs-up to the feature request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by MotionlessTrain** [comment]
This is already possible using the [setAbandoned](https://wiki.minecolonies.ldtteam.com/source/systems/command#colony-commands) subcommand. It would make you an officer of a colony, and you can start a new one. As long as you made sure you can still manage everything, you would effectively be managing two colonies at the same time.

Note that managing one colony is full-time already, so managing more is not easy at all

**Comment by Chinthor** [comment]
Agreed that the management is no small thing. This would be for late game, self-sustaining colonies. Or ones that are creative spawned in fully baked. I may not be understanding the setAbandoned feature. When I place the new town hall, it asks if I want to abandon the existing one, then kills all colonists if I say yes. Does using the command instead of the GUI not do this?

**Comment by Chinthor** [comment]
Did some more research and poking around. Seems there is a minor bug /w other dimension colonies that added to my confusion. I think I understand the abandon/officer thing now. I'll close this one and make a bug report for the other (much more minor) issue. Thanks for the reply!

**Comment by Chinthor** [comment]
One last question/clarification: how does this relate to the "easily abandon colonies: buggy" option in the config? Does setting that to true cause the GUI abandon to not kill colonists?

Sorry for coming back after I thought I could close.

**Comment by MotionlessTrain** [comment]
That option just changes an error/warning text iirc, to include the fact that you can abandon a colony before starting a new one
It doesn't change anything about the "delete" button on a town hall, nor does it change the setAbandoned command.

**Comment by uecasm** [comment]
The option in the town hall GUI is to delete the colony, not abandon it.  The command is the only way to abandon a colony.
